src/frontend/components/UploadManager/LTIUploadHandlers.tsx

```diff
@@ -15,18 +15,18 @@ const UploadSuccessHandler = ({
     if (objectState.status === UploadManagerStatus.SUCCESS) {
       const { file, objectId, objectType } = objectState;
       const object = await getResource(objectType, objectId);
-      if (object!.hasOwnProperty('title')) {
+      if ('title' in object! && !object.title) {
```

Comment:
Is there any reason why the  title is not set from the  backend during the upload ?

Reply:
Yes the uploaded file never reach the backend application. We directly upload on a S3 bucket using a policy upload.

Reply:
makes sense